### PR TITLE
deps(snyk): prettify snyk snapshot

### PR DESCRIPTION
**Pull request by patrickhulce**
should make future updates to snyk much more readable

### CODE REVIEW

File: lighthouse-core/scripts/cleanup-vuln-snapshot.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+writeFileSync(filename, output, 'utf8');
+
+/**
+ * @param {string} vulnString
+ * @return {string}
+ */
+function cleanAndFormat(vulnString) {
+  const snapshot = JSON.parse(vulnString);
+  for (const libEntries of Object.values(snapshot.npm)) {
+    libEntries.forEach((entry, i) => {
```
Comment (brendankenny) [inline lighthouse-core/scripts/cleanup-vuln-snapshot.js]: `lighthouse-core/scripts/` is type checked now, so can use the [`SnykDb` typedef from the audit](https://github.com/GoogleChrome/lighthouse/blob/2a20faba5394aae3f551aff51d2f57e49cf33090/lighthouse-core/audits/dobetterweb/no-vulnerable-libraries.js#L23) for the `JSON.parse` output.

File: lighthouse-core/scripts/cleanup-vuln-snapshot.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        severity: entry.severity,
+        semver: {vulnerable: entry.semver.vulnerable},
+      };
+
+      libEntries[i] = `SUPERHACKB${JSON.stringify(pruned)}SUPERHACKN`;
+    });
+  }
+
+  const out = JSON.stringify(snapshot, null, 2);
+  return out.replace(/"SUPERHACKB(.*?)SUPERHACKN"/g, function(_, s) {
```
Comment (brendankenny) [inline lighthouse-core/scripts/cleanup-vuln-snapshot.js]: any reason to use replace? The output looks like it ends up being just what you create in cleanAndFormat, so just use plain old `JSON.stringify()`?

Comment (brendankenny) [reply lighthouse-core/scripts/cleanup-vuln-snapshot.js]: oh, is this to get each entry on a single line? Agree with the "hack" part of the string, then :P

Comment (patrickhulce) [reply lighthouse-core/scripts/cleanup-vuln-snapshot.js]: hehe bingo, it goes from ~100 lines to ~600 lines, open to better ideas :)

Comment (paulirish) [reply lighthouse-core/scripts/cleanup-vuln-snapshot.js]: loooooooks like https://github.com/sergets/pretty-json-stringify is our dude.

**Comment by paulirish** [comment]
@patrickhulce how does https://github.com/GoogleChrome/lighthouse/pull/5080/commits/d4df09d64ff671e993967e8c0de09e05b610144c look?

**Comment by patrickhulce** [comment]
works for me @paulirish! 👀 👍

**Pull request merged**